### ISSUE TITLE
Remove border from .ui-widget-content css class

### ISSUE THREAD

**Issue by ChrisBr** [openSUSE/open-build-service#3820]
Issue/Feature description
====================
Introduced in ca1c351, the .ui-widget-content have now borders, see screenshot.

![datatable_border](https://user-images.githubusercontent.com/3799140/30377822-799e1e3e-9891-11e7-9dc8-e91bc68769c5.png)

Expected result
===============
There is no border anymore.

How to Reproduce
===============
https://build.opensuse.org/project/show/home:cbruckmayer

Further information
===============
Monkey patched in https://github.com/openSUSE/open-build-service/pull/3819

**Comment by ChrisBr** [comment]
Also what I just recognized is that the links in the datatables are also affected.

**Comment by hennevogel** [comment]
In general it's not a very good idea to change this globally. If you just need it for kiwi images, push this inside a class for them. Something like

```css
.kiwi_image .ui-widget-content {
  background: #ffffff;
  color: #222222;
}
```

**Comment by DavidKang** [comment]
Resolved in PR #3819 and PR #3826
